tests/data/TripAirReservationData.ts

```diff
@@ -0,0 +1,2566 @@
+                                    convertedCurrency: 'USD',
+                                    currencyCode: 'USD',
+                                    otherCoinage: [],
+                                },
+                                percent: 0,
+                            },
+                            conjunctionTicketSuffix: [],
+                            exchangePolicy: {
+                                exchangePenalty: {
+                                    amount: 149,
+                                    convertedAmount: 149,
+                                    convertedCurrency: 'USD',
+                                    currencyCode: 'USD',
+                                    otherCoinage: [],
+                                },
+                                isCat16: false,
+                                isConditional: false,
+                                isExchangeable: true,
+                            },
+                            fareCalculation: '',
+                            flightCoupons: [
+                                {
+                                    flightIdx: 0,
+                                    legIdx: 1,
+                                    status: 'NOT_FLOWN',
+                                },
+                                {
+                                    flightIdx: 0,
+                                    legIdx: 0,
+                                    status: 'NOT_FLOWN',
+                                },
+                            ],
+                            hemisphereCode: '',
+                            iataNumber: '45526666',
+                            issuedDateTime: {
+                                iso8601: '2025-10-13T11:57:00',
+                            },
+                            journeyCode: '',
+                            mcoIssuanceEligibility: 'NOT_ELIGIBLE',
+                            paymentDetails: [
+                                {
+                                    amount: {
+                                        base: {
+                                            amount: 136.15,
+                                            convertedAmount: 136.15,
+                                            convertedCurrency: 'USD',
+                                            currencyCode: 'USD',
+                                            otherCoinage: [],
+                                        },
+                                        tax: {
+                                            amount: 40.81,
+                                            convertedAmount: 40.81,
+                                            convertedCurrency: 'USD',
+                                            currencyCode: 'USD',
+                                            otherCoinage: [],
+                                        },
+                                    },
+                                    fop: {
```

Comment:
Can we please cut this data down to what is strictly needed in order to test the unit test? This also contains personal information from Alex which shouldn't be committed to the codebase since it is a public repository.